docs/aspnetcore-facility.md

```diff
@@ -124,6 +127,19 @@ public sealed class CustomMiddleware : IMiddleware
 }
 ```
 
+## Registering ASP.NET Core framework components in Windsor
+
+Simply put, don't do this. The instance that Windsor resolves might not always be the same instance that is resolved by the ASP.NET Core framework. If the 
+instance is stateful or is registered with a competing lifestyle this can lead to problems. 
+
+## Torn lifestyles
+
+If you register a framework component as scoped in Windsor and the ASP.NET Core framework has registered it as scoped also, two instances might appear. This
+problem is further obfiscated if those objects are stateful. This is known as a `torn lifestyle`. 
+
+References:
+ - https://simpleinjector.readthedocs.io/en/latest/tornlifestyle.html
+
```

Comment:
@jonorossi - this is what I have for `Torn Lifestyles` so far. I still need some time to think about `Captive Dependencies` if people choose to cross wire things.